swagger.yml

```
@@ -3022,9 +3022,13 @@ paths:
         - "Partner API v3"
       summary: List company inbox content
       operationId: GET_company-companyKey-content
-      description: >
+      description: |
         List the content this company has received.
         This may be filtered by certain `labels` and specific fields may be selected using `field`.
+
+        Since the list can be long, not all details about each content are in the response.
+        More detail is available via the
+        [Get specific content](#tag/Partner-API-v3/operation/Get%20content) endpoint.
```

```suggestion
        Note that more content metadata is available via the
        [Get specific content](#tag/Partner-API-v3/operation/Get%20content) endpoint.
```
Disclaimer reads weirdly imo.